src/components/MakeComment/MakeComment.jsx

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useState } from "react";
+import "./MakeComment.css";
+import { postComment } from "../../utils/app";
+import { useParams } from "react-router-dom";
+import CommentCard from "../CommentCard/CommentCard";
+
+const MakeComment = () => {
+
+  const params = useParams();
+  const article_id = params.article_id;
+
+  const [makeCommentState, setMakeCommentState] = useState(false);
+  const [currentUser, setCurrentUser] = useState("jessjelly");
+  const [commentBody, setCommentBody] = useState("");
+  const [commentComplete, setCommentComplete] = useState(false);
+  const [addedComment, setAddedComment] = useState("");
+
+ 
+  const makeComment = () => {
+    setMakeCommentState(true);
+  };
+
+ 
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setCommentComplete(true);
+    postNewComment(article_id, currentUser, commentBody);
```

Comment:
does this need to be a separate function? seems easier to just do the logic directly in here! - especially as the arguments dont actually need to be passed in to the function as the function is declared in the correct scope where it can access these variables/states